packages/liveview/src/lib.rs

```diff
@@ -109,3 +109,46 @@ pub fn interpreter_glue(url: &str) -> String {
     "#
     )
 }
+
+/// This function behaves similarly to [`interpreter_glue`], but interprets the provided URI as a
+/// relative path, automatically prefixing the protocol and host endpoint.
+///
+/// For instance, if you call `interpreter_glue("ws://localhost:1234/ws/liveview")` and the root of
+/// the liveview is being served at `localhost:1234`, you can replace it with
+/// `interpreter_glue_relative_uri("/ws/liveview")`.
+pub fn interpreter_glue_relative_uri(mut relative_uri: &str) -> String {
```

Comment:
The option to host the url on a relative path seems useful to make debug and production builds automatically point to a different host

Does this need to be a separate function from interpreter_glue? Or could we just check if the path start with a `/` (as you do bellow) and choose to interpret it as a relative or absolute URL from that.

Reply:
Oh, that's better, I'll fix the code in the direction you said and re-commit.

Reply:
I've merged `interpreter_glue_relative_uri` functionality into existing API, and editted `axum` example a bit to test if this works.